core/src/main/java/org/apache/iceberg/DeleteFileIndex.java

```diff
@@ -375,6 +402,31 @@ static Builder builderFor(Iterable<DeleteFile> deleteFiles) {
   }
 
   static class Builder {
+
+    private static final List<String> SCAN_COLUMNS =
```

Comment:
It would be nice to reuse these ones, we have similar vars in `BaseScan`.
I'll think more about it. Ideas are welcome.